Add FAQ about add-on creation needing personal information (#442)
CleverCloud/documentation: content/doc/find-help/faq.md

Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
Cellar has a replication system to protect the data stored inside. It creates three copies of your backups, each stored in a different datacenters in the PAR region.  
This way, even if one datacenter has an accident, your backup are still safe in two different datacenters.

## I can't create my add-on

> The creation of certain add-ons require you to fill the information of your account, such as the city, ZIP code, ect..  

Reviewer: ```suggestion
The creation of add-ons require you to fill the information of your account, such as the city, ZIP code, etc.
```

Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
Cellar has a replication system to protect the data stored inside. It creates three copies of your backups, each stored in a different data centers in the PAR region.  
This way, even if one data center has an accident, your backup are still safe in two different data centers.

## I can't create my add-on

> The creation of add-ons require you to fill the information of your account, such as the city, ZIP code, etc.

Reviewer: ```suggestion
To create add-ons, you need to complete your account information, including your city and ZIP code. 
```

> For example, you can't create a Matomo add-on if those information aren't present.

Reviewer: ```suggestion
For instance, you cannot create a Matomo add-on until you provide these details.```
